src/components/TopBarMenu/Cart/Cart.jsx

```diff
@@ -1,17 +1,51 @@
-import { useContext } from "react";
+import { useState, useContext } from "react";
 import CartContext from "./CartContext.js";
+import Images from "../../Images/Images.jsx";
+import Button from "../../Button/Button.jsx";
 
 import "./Cart.scss";
 
 export default function Cart() {
-  const { cart } = useContext(CartContext);
+  const { cart, setCart } = useContext(CartContext);
+  const [cartBox, setCartBox] = useState("cartboxOff");
+
+  function handleChangeCartbox() {
+    setCartBox(cartBox === "cartboxOff" ? "cartboxOn" : "cartboxOff");
+  }
+
+  function removeItemCart(cartQuantity) {
+    setCart({ product: null, quantity: 0, value: "false" });
+  }
 
   return (
-    <div className="teste">
-      <img src={`/images/${cart.product?.thumbnail}`} alt="" />
-      <p>{cart.product?.model}</p>
-      <p>{cart.quantity}</p>
-      <p>{cart.product?.price}</p>
-    </div>
+    <>
+      <Images
+        onClick={handleChangeCartbox}
+        source="icon-cart.svg"
+        imagetype="cart"
+        alt="cart icon"
+      />
+      <div id={cartBox}>
+        <h1>Cart</h1>
+        <div id={cart.value} className="cart-product">

```

Comment:
Always use classes with meaning,  true and false don't mean anything 🤔 
You can use variants for it.